<launch>
  <arg name="output_cmd_vel" default="arduino_cmd_vel"/>

  <arg name="teleop_cmd_vel"  default="teleop_cmd_vel"/>
  <arg name="processor_cmd_vel" default="processor_cmd_vel"/>
  <arg name="bumper" default="Bumper"/>

  <node pkg="ailab_mux" type="ailab_mux_node" name="ailab_mux_node" output="screen">
    <remap from="output_cmd_vel" to="$(arg output_cmd_vel)"/>
    <remap from="processor_cmd_vel" to="$(arg processor_cmd_vel)"/>
    <remap from="bumper" to="$(arg bumper)"/>
  </node>
</launch>